### PR TITLE
Plain timeline styler

### LINKED ISSUE #92: Plain Timeline View
Recently implemented a bubbled timeline styler. Add one more styler to use the plain one, like in Element.

### PR DESCRIPTION
**Pull request by ismailgulek**
Fixes #92

|Plain|Bubbles|Settings|
|-|-|-|
|Simulator Screen Shot - iPhone 13 Pro Max - 2022-06-24 at 15 27 39|Simulator Screen Shot - iPhone 13 Pro Max - 2022-06-24 at 15 27 56|Simulator Screen Shot - iPhone 13 Pro Max - 2022-06-24 at 16 14 40|

### CODE REVIEW

File: ElementX/Sources/Screens/Settings/View/Settings.swift [hunk 1]
```diff
@@ -85,6 +89,37 @@ struct Settings: View {
...
+                            .font(.body)
+                    }
+                }
+                .foregroundColor(Color.element.primaryContent)
+                .accessibilityIdentifier("timelineStyleButton")
+                .confirmationDialog(ElementL10n.settingsTimelineStyle,
+                                    isPresented: $showingTimelineStyles,
+                                    titleVisibility: .visible) {
+                    ForEach(TimelineStyler.allCases) { styler in
+                        Button { ElementSettings.shared.timelineStyle = styler.style
```
Comment (pixlwave) [inline ElementX/Sources/Screens/Settings/View/Settings.swift:112]: Not massively relevant given the comment about using a picker, but I wonder if this should be sending a view action. Might get tedious with lots of settings though 😕

**Review by pixlwave — COMMENTED** [review]
Well that didn't go away for long 😅 Comments inline.

**Review by pixlwave — APPROVED** [review]
Yep, looks good to me 😎

I would like to think/discuss about the architecture of settings storage a bit more, but that isn't necessary for this PR :)

File: ElementX/Sources/Other/ElementSettings.swift [hunk 2]
```diff
@@ -10,7 +10,7 @@ import Foundation
 import SwiftUI
 
 /// Store Element specific app settings.
-final class ElementSettings {
+final class ElementSettings: ObservableObject {
```
[the remark this answers is not in the document]
Comment (ismailgulek) [reply ElementX/Sources/Other/ElementSettings.swift:13]: We've discovered that after iOS 14.5 we can use `ObservableObject` with `@AppStorage` property wrappers like this. See https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-14_5-release-notes

**Pull request merged**